css/styles.css

```
@@ -479,6 +483,66 @@ body {
   margin-right: 10px;
 }
 
+.catalog-filter-list {
+  list-style-type: none;
+  margin: 0;
+  padding: 0;
+}
+
+.catalog-filter-item:not(:last-child) {
+  margin-bottom: 16px;
+}
+
+.control {
+  position: relative;
+  display: block;
+  padding-left: 36px;
+}
+
+.control-lable {
+  display: grid;
+  grid-template-columns: 20px 1fr;
+  gap: 16px;
+}
+
+.control-lable::before {
```

Конечно, все классы надо поправить:
```suggestion
.control-label::before {
```